src/components/slider/slider-utils.ts

```diff
@@ -0,0 +1,89 @@
+import type { DaikinSlider } from "./daikin-slider";
+
+/**
+ * This function removes trailing zeros from the decimal part and deletes the decimal point if it becomes the last character.
```

Comment:
Now I thought it would be better to make it more general-purpose.

```suggestion
 * Returns a normalized numerical representation.
 *
 * - Removes trailing zeros from the decimal part.
 * - Removes the decimal point if it becomes the last character.
 * - Removes leading zeros from the integer part.
```